src/tools/contrib/BalanceCPUCrimson.md

```diff
@@ -0,0 +1,59 @@
+# Balance CPU Crimson.
+
+----------
+
+We introduced the following utilities to help analysing the Performance impact of two strategies for
+allocation of CPU cores to Seastar reactor threads.
+
+- OSD-based: this consists on allocating CPU cores from the same NUMA socket to the same OSD.
+  for simplicity, if the OSD id is even, all its reactor threads are allocated to NUMA socket 0, and 
+  consequently if the OSD is is odd, all its reactor threads are allocated to NUMA socket 1.
+
+- NUMA socket based: this consists of allocating evenly CPU cores from each NUMA socket to the reactors, so 
+  all the OSD end up with reactor on both NUMA sockets.
+
+A new option `--crimson-balance-cpu <osd|socket>` has been implemented in `vstart.sh` to support these strategies.
+
+The utilities introduced are:
+
+- `balance-cpu.py`: a stand-alone script to produce the list of CPU core ids to use by `vstart.sh` when allocating
+  Seastar reactor threads. It uses as input the .json produced by `lscpu.py`.
+- `lscpu.py`: a Python module to parse the .json file created by `lscpu --json`. This produces a Python dictionary
+  with the NUMA details, that is, number of sockets, range of CPU core ids (physical and HT-siblings).
+- `tasksetcpu.py`: a stand-alone script to produce a grid showing the current CPU allocation, useful to quickly
+  visualise the allocation strategy.
+
+## Usage:
```

Comment:
Can you please show how to use each script added? in terms of expected parameters and so on.

Reply:
Sure, will add further details 👍